src/api.c

```diff
@@ -1752,6 +1774,33 @@ char *cg_build_path_locked(const char *name, char *path, const char *type)
 			if (name) {
 				char *tmp;
 
+				/*
+				 * If the user has passed absolute cgroup path,
+				 * avoid using systemd delegate slice as path
+				 * prefix.
+				 */
+				if (name[0] == '/') {
+					if (cg_namespace_table[i]) {
```

Comment:
Also, somewhat of a corner case, but tools like cgget used to allow the user to get settings in the root cgroup via `/` as the cgroup name.  How will that work here?